src/components/views/settings/DevicesPanel.js

```diff
@@ -222,9 +224,11 @@ export default class DevicesPanel extends React.Component {
                     <div className="mx_DevicesPanel_deviceId">{ _t("ID") }</div>
                     <div className="mx_DevicesPanel_deviceName">{ _t("Public Name") }</div>
                     <div className="mx_DevicesPanel_deviceLastSeen">{ _t("Last seen") }</div>
+                    {SettingsStore.getValue(UIFeature.SecurityAllowSessionEdits) &&
```

Comment:
Please remove all the user settings changes from this pull request

Reply:
I branched off from user setting so that when this is deployed to test, user setting plus branding can be reflected. Also, is user setting is still going to wait for sometimes more? I was thinking maybe we can merge user setting after branding. Also,  to avoid merge conflict between our changes I thought of getting latest code from user setting?

Reply:
You should always create your branch off master.  If this story was dependent on other stories (but this story isn't), then you could pull the files in from the other story branches that aren't yet in master, but that then means that the story can't go to master before the other stories are merged to master.

Reply:
No worries, I have reset branch to master's commit as base, and reapplied commit. This wont happen in future. I will always get branch from master.

Reply:
Please refer to pull request #12